library/base/src/main/java/org/cru/godtools/base/Settings.kt

```diff
@@ -51,7 +51,7 @@ class Settings @Inject internal constructor(
         const val FEATURE_TUTORIAL_ONBOARDING = "tutorialOnboarding"
         const val FEATURE_TUTORIAL_TRAINING = "tutorialTraining"
         const val FEATURE_TUTORIAL_LIVE_SHARE = "tutorialLiveShare."
-        const val FEATURE_TUTORIAL_TIPS = "tutorialTips"
+        const val FEATURE_TUTORIAL_TIPS = "tutorialTips."
```

Comment:
Why did you add a period here?

Reply:
because I'm concatenating the tool code to the end of the key. so the key will look like: `tutorialTips.kgp`

Reply:
Oh, gotcha!